addons.json

```diff
@@ -34,6 +36,7 @@
         "figshare",
         "github",
         "googledrive",
+	"owncloud",
```

Comment:
Huh. These three items were fixed in the last commit before I rebased to a single commit. I'm getting the sneaking suspicion that I dropped a commit when I rebased....